### DISCUSSION TITLE
[Bug]: Dependencies not Resolving Automatically in the New Version

### DISCUSSION THREAD

**Discussion by Quartzbell** [BerriAI/litellm#1508] [General]
### What happened?

In the earlier version (e.g., 1.7.9), the dependencies would be installed the first time a user used LiteLLM proxy. However, in the latest version, this no longer occurs:

<img width="577" alt="image" src="https://github.com/BerriAI/litellm/assets/6446747/571af7f1-74fe-4125-a0d5-b95df587a0b6">

**Comment by ghost** [comment]
Hi @Quartzbell it was an intentional decision, to solve runtime issues. 

We tried to simplify this by having all dependencies be installed via `pip install 'litellm[proxy]'` 

Let me know if you face issues with that.

**Reply by Quartzbell** [reply]
Thank you for your reply, @krrishdholakia.
I didn't notice that there is a change in the installation procedure.
Well, my automation and CI/CD suddenly stopped working. If this change is intended and there will be no automatic installation on the first run, then I will change the automation on my end.
Thank you for the good work.

**Reply by ishaan-jaff** [reply]
Hi @Quartzbell  thanks for this - can we hop on a call to better understand how we can improve litellm for you ? 

Sharing my calendly for your convenience: https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat
